src/nIsForName.js

```diff
@@ -22,7 +22,11 @@
  * @return {string[]}
  */
 function nIsForName(names) {
-  // write code here
+  const result = names.filter(elem =>
+    /\bN/.test(elem)
```

Comment:
if you want to place return value on the new line, you should wrap it in brackets